features/format.feature

```diff
@@ -458,11 +460,13 @@ Feature: Custom formats
         """
         And the content of file "2020-09-24_the-third-entry-finally-after-weeks-without-writing.md" in the cache should be
         """
+        ---
         title: The third entry finally after weeks without writing.
         date: 2020-09-24 09:14
         starred: False
         tags: tagone, tagthree
```

Comment:
This is missing the `body: |` part of the description in #1065. This is the main part of the yaml exporter than needs fixing. The `body` key needs to be output (followed by `: |`), then the body of the entry should follow on the next line, and be indented evenly, and indented farther than the `body` key.